Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Note that the indy-sdk requires ubuntu 16 and some custom dependencies so we can't use node:carbon-apline like the others
```

Comment:
Is Ubuntu 16 really still required version? I spot Docker file in indy-sdk using Ubuntu 18 https://github.com/hyperledger/indy-sdk/blob/master/libindy/ci/ubuntu18.dockerfile

Reply:
updated to 18.04